Source/GoBot/GoBotInternal.swift

```diff
@@ -182,7 +170,7 @@ class GoBotInternal {
                 
                 if worked {
                     self.replicate(feed: secret.identity)
-                    // make sure internal planetary pubs are authorized for connections
```

Comment:
This comment is a lie. This just makes us want to replicate data from those pubs.